designs/Website_V1.md

```diff
@@ -0,0 +1,11 @@
+# List of first re-design fixes:

```

Comment:
Can we have a link to the png images in this file? Otherwise the images are just staying orphan.